src/celestia/CelestiaCoreApplication.h

```diff
@@ -0,0 +1,159 @@
+#ifndef __CELESTIA_CORE_APPLICATION__H__ 
+#define __CELESTIA_CORE_APPLICATION__H__
+
+//#include <cstring>
+#include "celestiacore.h"
+
+class CelestiaCoreApplication : public CelestiaCore {
+    bool autoMessages;
+    
+public:
+    
+    CelestiaCoreApplication();
+    
+    void setAutoMessagesOn() { autoMessages = true; }
+    void setAutoMessagesOff() { autoMessages = false; }
+    bool getAutoMessages();
+    void aflash(const char *n) {
+        if (getAutoMessages())  flash(_(n));
+    }
+    
+    template <int F> class CelestiaRenderFlag {
+        int flag;
+        CelestiaCoreApplication *core;
+        const char *onMsg;
```

Comment:
It seems that you don't use `onMsg` & `offMsg` actually. `flashOn` & `flashOff`are never used.